completion/utilities.py

```diff
@@ -0,0 +1,18 @@
+#Python imports
+from __future__ import unicode_literals, absolute_import
+
+#Project imports
+from .models import BlockCompletion
+
+class UnavailableCompletionData(Exception):
+    def __init__(self):
+        Exception.__init__(self, "User's enrollment doesn't have completion data.")
+
+def get_key_to_last_completed_course_block(user, enrollment):
```

Comment:
Passing in the argument `enrollment` here is strange. If you are only planning to use the `course_id` out of the enrollment, please just pass in the `course_id` or `course_key`